packages/picasso.js/src/core/component/brushing.js

```diff
@@ -141,7 +141,11 @@ export function styler(obj, { context, data, style, filter, mode }) {
   const onUpdate = () => {
     const changed = update();
     if (changed) {
-      obj.renderer.render(obj.nodes);
+      if (typeof obj.config.sortNodes === 'function') {
+        obj.config.sortNodes(obj);
```

Comment:
It should be obj.renderer.render(obj.config.sortNodes(obj));

Reply:
Changed `obj.renderer.render` back to be in Picasso